esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -2974,7 +2974,7 @@ bool WsWuJobQueueAuditInfo::checkNewThorQueueItem(IEspThorQueue* tq, unsigned sh
             bAdd = true;
         if (!checkSameStrings(tq->getConnectedThors(), tq0.getConnectedThors()))
             bAdd = true;
-        if (!checkSameStrings(tq->getConnectedThors(), tq0.getConnectedThors()))
+        if (!checkSameStrings(tq->getWaitingThors(), tq0.getWaitingThors()))
```

Comment:
What is the purpose of this change?

Reply:
The old code is a duplicate of line 2975/2976. I guess that original author did a copy, but forgot to change it to getWaitingThors().